### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by mikecroft**
The idea here is to have some documentation on how to actually *use* MicroProfile, which is currently a gap that needs to be filled.

Ideally, each vendor could add documentation of how to use MicroProfile to this repository, since each will probably have their own hooks added to work with their servers in some way.

Right now, I've just linked Payara Micro to the official Payara Micro documentation as an example but, once we've added config 1.0 support, I would create a `getting-started/payara-micro` folder and keep all the documentation in there.

I think the most important thing for us here is to be **user friendly**. Even though it's a little more effort to add some (at least minimal) documentation specifically to this repository (rather than simply linking to official docs), I think it's worth it from the perspective of the confused user.

### CODE REVIEW

File: getting-started/getting-started.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+|===
+
+== Implementations
+
+Version 1.0 of MicroProfile is implemented by:
+
+* Apache TomEE
+* IBM Websphere Liberty
+* https://docs.payara.fish/documentation/payara-micro/payara-micro.html[Payara Micro]
+* WildFly Swarm
```
Comment (OndrejM) [inline getting-started/getting-started.adoc]: @MBJuric and @johnament, maybe you'd be also interested in listing KumuluzEE and Hammock? Please comment or raise a PR after this one is merged.

File: getting-started/getting-started.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
+= Getting Started with Eclipse MicroProfile
+
+The MicroProfile is a specification and does not provide a reference implementation. Many vendors already support MicroProfile and any other vendor is free to implement MicroProfile, or a specific API from MicroProfile.
```
Comment (OndrejM) [inline getting-started/getting-started.adoc]: I would first explain in more detail what MP is rather what it isn't. E.g.:
"The MicroProfile is a set of APIs and specifications developed within the Eclipse foundation by many parties from the Java community. It doesn't provide any reference implementation but specifies the API and behavior of MicroProfile implementations. It also provides basic information to get started with particular implementations, which already support MicroProfile. Any other vendor is free to implement MicroProfile or a specific APi from MicroProfile."

Comment (mikecroft) [inline getting-started/getting-started.adoc]: Yes, perhaps the "does not provide a reference implementation" bit should be moved to the bit below where implementations are listed?

File: getting-started/getting-started.adoc (hunk 1 above)
Comment (MBJuric) [inline getting-started/getting-started.adoc]: Yes, please. We are interested to be listed: https://ee.kumuluz.com/ [KumuluzEE]
We provide a fully compliant MicroProfile 1.0 implementation. Thanks Ondrej.

Comment (mikecroft) [inline getting-started/getting-started.adoc]: Added

File: getting-started/getting-started.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
+= Getting Started with Eclipse MicroProfile
+
+The MicroProfile is a set of APIs and specifications developed within the Eclipse foundation by many parties from the Java community. It doesn't provide any reference implementation but specifies the API and behavior of MicroProfile implementations. It also provides basic information to get started with particular implementations, which already support MicroProfile. Any other vendor is free to implement MicroProfile or a specific API from MicroProfile.
+
+== Creating a MicroProfile project with Maven
+
+Version 1.0 of MicroProfile was released before the project was moved to be under the Eclipse Foundation, so the Maven coordinates are a little different. To create a project which targets version 1.0 of MicroProfile - including JAX-RS 2.0, CDI 1.1 and JSON-P 1.0 - you should include the following dependency:
+
+[source, xml]
+---
```
Comment (mikecroft) [inline getting-started/getting-started.adoc]: should be 4 dashes here

**Comment by kwsutter** [comment]
Since there has been no activity with this PR for 1-1/2 years, I'm going to close it.  If there is strong desire to still merge this content, re-open and re-fresh the PR.  Thanks!

**Pull request closed without merging**